pkg/workceptor/kubernetes.go

```diff
@@ -295,7 +295,14 @@ func (kw *kubeUnit) runWorkUsingLogger() {
 		Container: "worker",
 		Follow:    true,
 	})
-	logStream, err := logreq.Stream(kw.ctx)
+	var logStream io.ReadCloser
+	for retries := 0; retries < 5; retries++ {
+		logStream, err = logreq.Stream(kw.ctx)
+		if err != nil {
```

Comment:
Shouldn't we be breaking out of this loop early if err is nil?

Reply:
you're right, missing a break